app_server/public/js/modules/components/datadisplay.js

```diff
@@ -0,0 +1,49 @@
+const m = require('mithril');
+
+const ItemsCount = require('../components/itemscount');
+const Pagination = require('../components/pagination');
+const Datatable = require('../components/datatable');
+
+const Datadisplay = module.exports = {};
+
+Datadisplay.controller = function () {
+  let ctrl = this;
+
+  ctrl.currentPage = m.prop(m.route.param('page') || 1);
+  ctrl.itemsPerPage = m.prop(m.route.param('items') || 10);
+  ctrl.maxPage = m.prop(1);
+
```

Comment:
Not sure that the literal numbers `1` and `10` mean here. If they are temporary default values then should be fine, otherwise, named constants should be more readable.